packages/core/src/App/Containers/RealAccountSignup/add-or-manage-accounts.jsx

```diff
@@ -94,94 +94,102 @@ class AddOrManageAccounts extends React.Component {
-                                    })}
-                                >
-                                    {!this.props.can_change_fiat_currency && (
-                                        <div className='account-wizard--disabled-message'>
-                                            <p>
-                                                {this.props.current_currency_type === 'fiat' ? (
-                                                    <Localize
-                                                        i18n_default_text='Currency change is not available because either you have deposited money into your {{currency}} account or you have created a real MetaTrader 5 (MT5) account.'
-                                                        values={{
-                                                            currency: getCurrencyDisplayCode(this.props.currency),
-                                                        }}
-                                                    />
-                                                ) : (
-                                                    <Localize
-                                                        i18n_default_text='Please switch to your {{fiat_currency}} account to change currencies.'
-                                                        values={{
-                                                            // eslint-disable-next-line
-                                                            fiat_currency: this.props.current_fiat_currency.toUpperCase(),
-                                                        }}
-                                                    />
-                                                )}
-                                            </p>
-                                        </div>
-                                    )}
-                                    <ChangeAccountCurrency
-                                        className='account-wizard__body'
-                                        onSubmit={this.updateValue}
-                                        value={this.state.form_value}
-                                        form_error={this.state.form_error}
-                                        {...this.props}
-                                    />
-                                </div>
-                            ) : (
+                )}
+                <ChangeAccountCurrency
+                    className='account-wizard__body'
+                    onSubmit={this.updateValue}
+                    value={this.state.form_value}
+                    form_error={this.state.form_error}
+                    {...this.props}
+                />
+            </div>
+        );
+
+        return (
+            <ThemedScrollbars is_bypassed={isMobile()} autohide={false}>
+                {this.props.is_eu && this.props.has_fiat ? (
+                    fiat_section
```

Comment:
Since you're rendering a component, can we use JSX instead?
`<fiat_section />`

Reply:
No. It's not a component. It's just a variable holding some JSX.

Reply:
yeah, I know, since it goes to the render method, it's nicer to use it like that.